### DISCUSSION TITLE
Notification language

### DISCUSSION THREAD

**Discussion by baptistejstn** [sct/overseerr#3635] [Feature Requests]
Hi,

The ability to set default language for the whole application is very good, except for notifications that aren't tranlated. Is it a bug, being implemented or simply forgotten ? Will it be modified in the next months ?

Baptiste.

**Comment by TheCatLady** [comment]
Not a bug. Server-side translations are not yet supported. They are planned, but no ETA.

**Reply by EosReborn** [reply]
It would be great if this were to happen🙏🙏

**Comment by dtalens** [comment]
Please, add support for translate notifications.
